sonar-ws/src/main/java/org/sonarqube/ws/client/WsRequest.java

```diff
@@ -31,16 +34,25 @@
   private MediaType mimeType = MediaType.JSON;
   private String url;
 
-  public WsRequest(String url) {
+  private WsRequest(String url) {
```

Comment:
Param `url` should be renamed to something like `endpoint`, as expected value is "api/issues/search" but not "http://localhost:9000/api/issues/search"